show/main.py

```diff
@@ -129,6 +173,148 @@ def run_command(command, display_cmd=False):
     if rc != 0:
         sys.exit(rc)
 
+
+def get_interface_mode():
+    mode = os.getenv('SONIC_CLI_IFACE_MODE')
+    if mode is None:
+        mode = "default"
+    return mode
+
+
+# Global Alias object
+_interface = InterfaceAliasConverter()
+
+
+def print_output_in_alias_mode(output, index):
+    """Convert and print all instances of SONiC interface
+       name to vendor-sepecific interface aliases.
+    """
+
+    alias_name = ""
+    interface_name = ""
+
+    # Adjust tabulation width to length of alias name
+    if output.startswith("---"):
+        word = output.split()
+        dword = word[index]
+        underline = dword.rjust(_interface.vendor_name_max_length, '-')
+        word[index] = underline
+        output = '  ' .join(word)
+
+    # Replace default interface name to vendor-name
```

Comment:
```
# Global class instance for SONiC interface name to alias conversion
convert_interface = InterfaceAliasConverter()
```
Renamed as convert_interface so, if name_to_alias / alias_to_name functions are used along with this variable name it will be more meaningful..

```
if get_interface_mode() == "alias":
1174                     alias = convert_interface.name_to_alias(m)
1175                     r.append(alias)
1176                 else:
1177                     r.append(m)
```